platform/viewer/src/routes/WorkList/WorkList.tsx

```diff
@@ -345,12 +345,17 @@ function WorkList({
             // mode.routeName
             // mode.routes[x].path
             // Don't specify default data source, and it should just be picked up... (this may not currently be the case)
-            // How do we know which params to pass? Today, it's just StudyInstanceUIDs
+            // How do we know which params to pass? Today, it's just StudyInstanceUIDs and configUrl if exists
+            const query = new URLSearchParams();
+            if (filterValues.configUrl) {
+              query.append('configUrl', filterValues.configUrl);
+            }
+            query.append('StudyInstanceUIDs', studyInstanceUid);
             return (
               <Link
                 key={i}
                 to={`${dataPath ? '../../' : ''}${mode.routeName}${dataPath ||
-                  ''}?StudyInstanceUIDs=${studyInstanceUid}`}
+                  ''}?${decodeURIComponent(query.toString())}`}
```

Comment:
This fails for the situation where the configUrl includes parameters, such as:
configUrl=https://myhost/config?a=b&c=d
because the & is encoded/decoded.  The decodeURIComponent, as the name says, needs to apply to individual components.

Reply:
@wayfarer3130  Do you mean decodeURIComponent should be removed and original queryString should be passed?